models/gc_spider_baseline/meta.json

```diff
@@ -0,0 +1,135 @@
+    "inputs": [
+      {
+        "label": "Sagittal Spine MRI",
+        "description": "Sagittal MRI of the spine",
+        "format": "DICOM",
+        "modality": "MR",
+        "bodypartexamined": "Spine",
+        "slicethickness": "2.0mm",
+        "non-contrast": false,
+        "contrast": false
+      }
+    ],
+    "outputs": [
+      {
+        "type": "Segmentation",
+        "classes": [
+          "VERTEBRAE_L5",
+          "VERTEBRAE_L4",
+          "VERTEBRAE_L3",
+          "VERTEBRAE_L2",
+          "VERTEBRAE_L1",
+          "VERTEBRAE_T12",
+          "VERTEBRAE_T11",
+          "VERTEBRAE_T10",
+          "VERTEBRAE_T9",
+          "VERTEBRAE_T8",
+          "VERTEBRAE_T7",
+          "VERTEBRAE_T6",
+          "VERTEBRAE_T5",
+          "VERTEBRAE_T4",
+          "VERTEBRAE_T3",
+          "VERTEBRAE_T2",
+          "VERTEBRAE_T1",
+          "VERTEBRAE_C7",
+          "VERTEBRAE_C6",
+          "VERTEBRAE_C5",
+          "VERTEBRAE_C4",
+          "VERTEBRAE_C3",
+          "VERTEBRAE_C2",
+          "VERTEBRAE_C1"
+        ]
+      }
+    ],
+    "model": {
+      "architecture": "3D U-Net-like fully-convolutional neural network",
+      "training": "supervised",
+      "cmpapproach": "2D"
+    },
+    "data": {
+      "training": {
+        "vol_samples": 179
+      },
+      "evaluation": {
+        "vol_samples": 39
+      },
+      "public": false,
+      "external": false
+    }
+  },
+  "details": {
+    "name": "SPIDER Baseline algorithm: Iterative Instance Segmentation of the spine",
+    "version": "",
+    "devteam": "DIAGNijmegen (Diagnostic Image Analysis Group, Radboud UMC, The Netherlands)",
+    "type": "U-net",
+    "date": {
+      "weights": "",
+      "code": "2023-08-15",
+      "pub": "2023-06-23"
+    },
+    "cite": "Nikolas Lessmann, Bram van Ginneken, Pim A. de Jong, Ivana Išgum, Iterative fully convolutional neural networks for automatic vertebra segmentation and identification, Medical Image Analysis, Volume 53, 2019, Pages 142-155, ISSN 1361-8415, https://doi.org/10.1016/j.media.2019.02.005.",
+    "license": {
+      "code": "Apache 2.0",
+      "weights": "Apache 2.0"
+    },
+    "publications": [
+      {
+        "title": "Lumbar spine segmentation in MR images: a dataset and a public benchmark",
+        "uri": "https://arxiv.org/abs/2306.12217"
+      }
+    ],
+    "github": "https://github.com/DIAGNijmegen/SPIDER-Baseline-IIS",
+    "zenodo": "",
+    "colab": "",
+    "slicer": false
+  },
+  "info": {
+    "use": {
+      "title": "Intended use",
+      "text": "Segmentation of three anatomical structures in lumbar spine MRI: vertebrae, intervertebral discs (IVDs), and spinal canal.",
```

Comment:
This describes what the algorithm is doing (redundand to description) not how it is intended to be used.